Tests/FluentAssertions.Specs/Specialized/TaskOfTAssertionSpecs.cs

```diff
@@ -54,6 +55,53 @@ public async Task When_task_completes_fast_async_it_should_succeed()
             await action.Should().NotThrowAsync();
         }
 
+        [Fact]
+        public async Task When_task_completes_async_and_result_is_not_expected_it_should_throw()
+        {
+            // Arrange
+            var timer = new FakeClock();
+            var taskFactory = new TaskCompletionSource<int>();
+
+            // Act
+            Func<Task> action = async () =>
+            {
+                Func<Task<int>> funcSubject = () => taskFactory.Task;
+
+                (await funcSubject.Should(timer).CompleteWithinAsync(100.Milliseconds()))
+                    .Which.Should().Be(42);
+            };
+
+            taskFactory.SetResult(99);
+            timer.Complete();
+
+            // Assert
+            // TODO message currently shows "Expected (await funcSubject to be...", but should be "Expected funcSubject to be...",
```

Comment:
While having deep look on the generated failure message I found the following current result:
`Expected (await funcSubject to be...`
The brace should be removed, isn't it @dennisdoomen , @jnyrup ?
Should we remove `await` too?

What should be the correct messge?
`Expected await funcSubject to be...` or `Expected funcSubject to be...`

This should be tracked separately!?

Reply:
You could replace `await` with 'awaitable' to make it more fluent and keep it distinct from a non-awaited execution?

Reply:
I like the suggestion @eNeRGy164 made.

Reply:
@eNeRGy164, you mean the generated error message should contain "awaitable" instead of "await"!?

Reply:
Yes, so to pick your example

`Expected awaitable funcSubject to be...`

Reply:
I've moved this issue to #1488.